### PR TITLE
bootstrap_urls in bastille.conf

### PR DESCRIPTION
**Pull request by hackacad**
configure bootstrap url for FreeBSD and Hardened in bastille.conf
will fallback to official FreeBSD URL on failure

### CODE REVIEW

**Comment by JRGTH** [comment]
I like this too, please also check the `bastille.conf` indentation as bastille use spaces rather than tabs to keep code consistency/layout.

Regards

**Comment by hackacad** [comment]
Definitely. 
Another point is, whether to add more rows (especially for the hardened bootstrap url) or adding a new line for this annotation.

```
## bootstrap urls
bastille_url_freebsd="http://ftp.freebsd.org/pub/FreeBSD/releases/"                     ## default: "http://ftp.freebsd.org/pub/FreeBSD/releases/"
bastille_url_hardenedbsd="https://installer.hardenedbsd.org/pub/HardenedBSD/releases/"  ## default: "https://installer.hardenedbsd.org/pub/HardenedBSD/releases/"
```
I'll convert it and upload it again.

Maybe it's worth thinking about removing the annotations completely or at least for the HardenedBSD line.

File: usr/local/share/bastille/bootstrap.sh [hunk 1]
```diff
@@ -404,7 +404,7 @@ fi
 NAME_VERIFY=$(echo "${RELEASE}" | grep -iwE '^([1-9]{2,2})(-stable-LAST|-STABLE-last|-stable-last|-STABLE-LAST)$' | sed 's/STABLE/stable/g' | sed 's/last/LAST/g')
 if [ -n "${NAME_VERIFY}" ]; then
     RELEASE="${NAME_VERIFY}"
-    UPSTREAM_URL="https://installer.hardenedbsd.org/pub/HardenedBSD/releases/${HW_MACHINE}/${HW_MACHINE_ARCH}/hardenedbsd-${RELEASE}"
+    UPSTREAM_URL="i${bastille_url_hardenedbsd}/${HW_MACHINE}/${HW_MACHINE_ARCH}/hardenedbsd-${RELEASE}"
```
Comment (cedwards) [inline usr/local/share/bastille/bootstrap.sh]: extra "i" character?

Comment (hackacad) [reply usr/local/share/bastille/bootstrap.sh]: fixed

**Comment by cedwards** [comment]
Thank you.

**Pull request merged**